### PR TITLE
add `nu_plugin_image` plugin

### PR DESCRIPTION
**Pull request by FMotalleb**
adds two commands 
* `from png`: open png files in terminal 
* `to png`: converts ansi outputs `colored texts, tables, etc` into image file

### CODE REVIEW

File: plugin_details.md
Comment (amtoine) [inline plugin_details.md:1]: do you happen to know why this file is changing all of a sudden? :thinking:

**Comment by FMotalleb** [comment]
I added a null check where (I think) was the cause of the issue and from the output it most likely is
```
EvalBlockWithInput(Span { start: 107690, end: 107694 }, [CantFindColumn { col_name: "plugin", span: Span { start: 114641, end: 114647 }, src_span: Span { start: 114634, end: 114640 } }])
```

**Comment by fdncred** [comment]
I'm fine with landing this but I think all those eprintln messages should be put behind a `--verbose` flag.

**Comment by FMotalleb** [comment]
> I'm fine with landing this but I think all those eprintln messages should be put behind a `--verbose` flag.

done

**Comment by fdncred** [comment]
Thanks. Looks good. I really like this plugin.

**Comment by FMotalleb** [comment]
Sadly, for now time is tight but Ill update `from png` and fix its low resolution issue

**Pull request merged**